main-settings/src/main/scala/sbt/Scope.scala

```diff
@@ -408,9 +408,9 @@ object Scope {
       projectInherit: ProjectRef => Seq[ProjectRef],
       configInherit: (ResolvedReference, ConfigKey) => Seq[ConfigKey]
   ): DelegateIndex = {
-    val pDelegates = refs map { case (ref, project) =>
+    val pDelegates = (refs.map { case (ref, project) =>
       (ref, delegateIndex(ref, configurations(project))(projectInherit, configInherit))
-    } toMap;
+    }).toMap
```

Comment:
I don't think we need parentheses around `refs.map()`.

Also maybe this is a bit more readable:
```scala
    val pDelegates = refs
      .map { case (ref, project) => 
        (ref, delegateIndex(ref, configurations(project))(projectInherit, configInherit))
      }
      .toMap
```